data/json/items/comestibles/raw_veggy.json

```diff
@@ -181,6 +181,27 @@
     "smoking_result": "dry_veggy",
     "vitamins": [ [ "vitC", 2 ], [ "calcium", 4 ], [ "iron", 2 ], [ "veggy_allergen", 1 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "radish_taproot",
+    "name": { "str_sp": "radish taproot" },
+    "weight": "100 g",
+    "color": "red",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "//": "No idea what else to suggest here",
```

Comment:
This is not a useful doc comment, please remove. (It's absolutely 100% fine and great in the github PR, just not in the persistent code)